content/en/docs/reference/glossary/resource.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Resource
+id: resource
+date: 2025-02-09
+full_link: https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology
+short_description: >
+  A Kubernetes entity, representing a RESTful endpoint on the Kubernetes API server
```

Comment:
Kubernetes' API architecture is heavily inspired by ReST, but we can't be sure that all resources are part of ReST-like APIs. For example, Pods have various subresources that are more like RPC or streaming services (they don't have defined representations).

Reply:
fair, I think best to simply remove "RESTful" then.